[2025-04-27 10:51:54,570](__post_init__) Проблема с подключением к серверу очереди: HTTPConnectionPool(host='192.168.100.8', port=32107): Max retries exceeded with url: /get_wplace?w=8 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001B94397F250>: Failed to establish a new connection: [WinError 10061] Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение'))
